PSYA/donations.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- ticker: PSYA
+  recipient: 
+    name: MAPS, Multidisciplinary Association for Psychedelic Studies
+    address: 3141 Stevens Creek Blvd 40563, San Jose, CA 95117
+    tel: +1 831-429-MDMA (6362)
+    contact: giving@maps.org
+  ada: 137 # <- These must be numbers. Can be decimals though. See below
+  usd: 192 # Only usd currency accepted, check on xe.com for exchange rate.
+  timestamp: 1625569261 
+  files:
+    - PSYA/first.jpg
+    - PSYA/first_LTC.pjpg
+  announcement_url: https://twitter.com/psyadapool/status/1412525129087737863
+  # charity_tx_id: c9a866dd665285299236164805bfdeb8e46ac20cee76407ba2bd0e2fe553b133
+  intermediate_txs:
```

Comment:
Hi @rafek47 !
I understood that a payment to the LTC Adress sLMpTFx2ts5mVPjhpqQ5oYCs7zuq2AHVJES happened.
How do I see that this address belongs to maps?
Do you have some receipt or confirmation from maps as well?
Regards, 
Markus

Reply:
Hi, there is no confirmation from them, they don't have anything like that. There is only LTC address on their webpage. https://store.maps.org/np/clients/maps/neonPage.jsp?pageId=19&